### PR TITLE
Modified kickstart to have only one instance of BOOTPROTo and ONBOOT …

### PR DESCRIPTION
**Pull request by kostew**
Review the changes to centos-ks to allow interfaces that do not have DHCP.

@RackHD/rackhd_dev @RackHD/corecommitters @johren 
@jimturnquist

### CODE REVIEW

**Comment by yyscamper** [comment]
:+1:

**Comment by anhou** [comment]
:+1:

File: data/templates/centos-ks [hunk 1]
```diff
@@ -104,6 +104,8 @@ echo -e "NETWORKING=yes\nHOSTNAME=<%=hostname%>.<%=domain%>" > /etc/sysconfig/ne
       <% if( undefined != n.ipv4.vlanId ) { %>
         <% n.ipv4.vlanId.forEach(function(vid) { %>
           echo "Configuring vlan <%=vid%> on <%=n.device%>"
```
Comment (jimturnquist) [inline data/templates/centos-ks]: These sed commands aren't operating on the correct file. They should be operating on <%=n.device%>.<%=vid%>

File: data/templates/centos-ks [hunk 2]
```diff
@@ -126,6 +130,8 @@ echo -e "NETWORKING=yes\nHOSTNAME=<%=hostname%>.<%=domain%>" > /etc/sysconfig/ne
       <% if( undefined != n.ipv6.vlanId ) { %>
         <% n.ipv6.vlanId.forEach(function(vid) { %>
           echo "Configuring vlan <%=vid%> on <%=n.device%>"
+          sed -i '/^BOOTPROTO=/d' /etc/sysconfig/network-scripts/ifcfg-<%=n.device%>
```
Comment (jimturnquist) [inline data/templates/centos-ks]: Same comment here, should be /etc/sysconfig/network-scripts/ifcfg-<%=n.device%>.<%=vid%>

**Comment by anhou** [comment]
test this please

**Comment by anhou** [comment]
test this please

**Comment by diontje** [comment]
@RackHD/corecommitters @RackHD/rackhd_dev @johren @jimturnquist @frostr3 
Is there anything preventing this merge?
This PR is currently blocking 20+ OnRack v1.2 tests.

**Comment by heckj** [comment]
@diontje Jim's comment sed commands not being appropriate stand out to me...

**Comment by kostew** [comment]
Jim's sed comments have been corrected and code resubmitted, see above.

**Comment by heckj** [comment]
Ah - kk - looks good to merge to me then.

**Comment by diontje** [comment]
perfect. Thanks!

**Pull request merged**